Makefile

```diff
@@ -22,6 +22,9 @@ test-before-build:
 	## Check that all slugs are unique
 	! git --no-pager grep -h "^slug: " _posts | sort | uniq -d | grep .
 
+	## Check that newly added or modifyed PNGs are optimized
+	sh _contrib/travis-check-png-optimized.sh
```

Comment:
`sh` needed? Could do `_contrib/travis-check-png-optimized.sh` assuming it is executable?